### PR TITLE
Feature/solidity voting review comments

### PR DESCRIPTION
**Pull request by ronnno**
- function naming:
```
getRegistrationBlockHeight -> getRegistrationBlockNumber
addValidator -> approve
getApprovalBockHeight -> getApprovalBlockNumber
getRegistrationBlockHeight -> getRegistrationBlockNumber // (guardians and validators registry)
getLastVote -> getCurrentVote
```
in various places (parameter names): `blockHeight -> blockNumber`

- separate getters for lists of addresses to address version and bytes20 version
   - changed return type to `address[]`: `getValidators`, `getGuardians`
   - added with return type `bytes20[]`: `getGuardiansBytes20`, `getValidatorsBytes20`

- OrbsValidators.isApproved() made public
- minor bug fixes and review comments

### CODE REVIEW

File: voting/ethereum/contracts/OrbsGuardians.sol [hunk 1]
```diff
@@ -62,14 +62,18 @@ contract OrbsGuardians is IOrbsGuardians {
 
         uint i = guardiansData[msg.sender].index;
 
-        assert(guardians[i] == msg.sender);
+        assert(guardians[i] == msg.sender); // will consume all available gas.
 
+        // replace with last element and remove from end
         guardians[i] = guardians[guardians.length - 1]; // switch with last
         guardiansData[guardians[i]].index = i; // update it's lookup index
+        delete guardians[guardians.length - 1]; // remove the last one
```
Comment (avilanthe1) [inline voting/ethereum/contracts/OrbsGuardians.sol]: I think it is redundant to delete the last element if the array's length is decreased any how.
See further details in [this comment](https://ethereum.stackexchange.com/questions/1527/how-to-delete-an-element-at-a-certain-index-in-an-array#comment43646_1528).

File: voting/ethereum/contracts/OrbsValidators.sol [hunk 2]
```diff
@@ -41,76 +41,94 @@ contract OrbsValidators is Ownable, IOrbsValidators, IOrbsNetworkTopology {
...
     function remove(address validator) public onlyOwner {
+        require(isApproved(validator), "Not an approved validator");
+
         uint approvedLength = approvedValidators.length;
         for (uint i = 0; i < approvedLength; ++i) {
             if (approvedValidators[i] == validator) {
+
+                // replace with last element and remove from end
                 approvedValidators[i] = approvedValidators[approvedLength - 1];
                 delete approvedValidators[approvedLength - 1];
```
Comment (avilanthe1) [inline voting/ethereum/contracts/OrbsValidators.sol]: Same as above

File: voting/ethereum/contracts/OrbsGuardians.sol (hunk 1 above)
Comment (ronnno) [reply voting/ethereum/contracts/OrbsGuardians.sol]: fixed in another branch

File: voting/ethereum/contracts/OrbsValidators.sol (hunk 2 above)
Comment (ronnno) [reply voting/ethereum/contracts/OrbsValidators.sol]: fixed in another branch

**Pull request merged**